components/post_view/post_list/post_list_virtualized.jsx

```diff
@@ -336,6 +339,16 @@ export default class PostList extends React.PureComponent {
             }
         }
 
+        if (scrollUpdateWasRequested) { //if scroll change is programatically requested i.e by calling scrollTo
+            //This is a private method on virtlist
+            const postsRenderedRange = this.listRef.current._getRangeToRender(); //eslint-disable-line no-underscore-dangle
+
+            // postsRenderedRange[3] is the visibleStopIndex which is post at the bottom of the screen
+            if (postsRenderedRange[3] <= 1 && canLoadNewerPosts) {
```

Comment:
Why check if the stop index is <= 1 specifically? Why don't we check if it's 0?

Reply:
No particular reason i thought it just gives a little buffer for eager loading the next API